content/05.Experimental_Design.md

```diff
@@ -2,14 +2,19 @@
 
 <!-- Leader: Will Lewis; co-leaders: James Martin -->
 
-## Repetition 
+## Randomization 
+This section will discuss the importance of randomly assigning treatments to groups in experimental studies. The treatment is not ranomly applied to groups in observational studies, though methods such a random selection of study sites and survey areas can help to reduce bias. Some of this may overlap with previous sections describing the difference between experimental and observational studies.
```

Comment:
```suggestion
This section will discuss the importance of randomly assigning treatments to groups in experimental studies. The treatment is not randomly applied to groups in observational studies, though methods such a random selection of study sites and survey areas can help to reduce bias. Some of this may overlap with previous sections describing the difference between experimental and observational studies.
```